examples/cmake/thirdparty/fetch_rapids.cmake

```diff
@@ -14,8 +14,8 @@
 # Use this variable to update RAPIDS and RAFT versions
```

Comment:
Should this comment be updated, too?
```suggestion
# Use this variable to update RAPIDS and cuVS versions
```

Reply:
I think it should be updated, since changing this variable will change the cuVS version being used.